packages/hash/api/src/index.ts

```diff
@@ -132,13 +137,20 @@ const main = async () => {
     port: graphApiPort,
   });
 
+  await ensureSystemOrgAccountIdExists({ graphApi, logger });
+
   await ensureSystemTypesExist({ graphApi, logger });
 
+  await ensureSystemOrgExists({
+    graphApi,
+    logger,
+  });
+
   await ensureSystemEntitiesExists({ graphApi, logger });
 
   // This will seed users, an org and pages.
```

Comment:
This wasn't updated

Reply:
I'm not entirely sure what you're referring to, the comment? In which case this was removed